.travis.yml

```diff
@@ -19,7 +19,6 @@ before_script:
   - composer self-update
   - composer install
   - ./bin/jackrabbit.sh
-  - php bin/jackrabbit.php
```

Comment:
Its the travis file in the search bundle which was already out of date.

Better to add .travis.yml files if and when we need them.

On Tue, Mar 01, 2016 at 06:17:45AM -0800, Daniel Rotter wrote:
>    In [1].travis.yml:
> 
>  > @@ -19,7 +19,6 @@ before_script:
>  >    - composer self-update
>  >    - composer install
>  >    - ./bin/jackrabbit.sh
>  > -  - php bin/jackrabbit.php
> 
>    Why is this removed?
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/2076#discussion_r54571673
>    2. https://github.com/sulu-io/sulu/pull/2076/files#r54571673
>    3. https://github.com/sulu-io/sulu/pull/2076/files#r54571673